src/test/java/cloud/eppo/BaseEppoClientTest.java

```diff
@@ -311,5 +334,32 @@ public void testAssignmentLogErrorNonFatal() {
     verify(mockAssignmentLogger, times(1)).logAssignment(assignmentLogCaptor.capture());
   }
 
+  @Test
+  public void testConfigurationStoreOverride() throws IOException {
```

Comment:
What is this test testing exactly? That if the configuration changes after initialization we get updated assignments?

Reply:
It's testing that the client is actually using the `configurationStoreOverride argument. Not really necessary, I suppose. When the ConfigStore has more capabilities, those tests will rely on being able to pass in a ConfigStore (the android SDK will also prove this out).